skyportal/handlers/api/spectrum.py

```diff
@@ -133,6 +133,8 @@ def post(self):
         spec.instrument = instrument
         spec.groups = groups
         spec.owner_id = owner_id
+        if spec.type is None:
+            spec.type = 'source'
```

Comment:
Don't hard code this since, it could be difference in different configs:
`spec.type = cfg.get('spectrum_types.default', "source")`